lib/queries/organizationQueries.ts

```
@@ -0,0 +1,434 @@
+        console.error(`Error saving member ${member.name}:`, error)
+        throw error
+      }
+    })
+  )
+}
+
+// Helper function to generate and save products
+async function generateAndSaveProducts(
+  organization: { id: string; name: string; description: string | null },
+  count: number
+): Promise<Prisma.ProductGetPayload<{}>[]> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)